<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>socratesClass</class>
 <widget class="QMainWindow" name="socratesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>socrates</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <item>
     <layout class="QFormLayout" name="controlLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="portCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileLabel">
        <property name="text">
         <string>Soubor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="fileLayout" stretch="1,0">
        <item>
         <widget class="QLineEdit" name="fileLine">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileBrowseButton">
          <property name="text">
           <string>Procházet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Délka experimentu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="timeLayout">
        <item>
         <widget class="QCheckBox" name="stopCheckbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Zastavit po</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="inputMask">
           <string>00:00:00</string>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="startStopButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="resultsLayout" stretch="1,0,0">
      <item>
       <widget class="QTableWidget" name="resultTableWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="totalLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>Celkový čas</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="gate1desc">
          <property name="text">
           <string>Brána 1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="gate2desc">
          <property name="text">
           <string>Brána 2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="gate3desc">
          <property name="text">
           <string>Brána 3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="gate4desc">
          <property name="text">
           <string>Brána 4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="gate1label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="gate2label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="gate3label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="gate4label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="socrates.qrc"/>
 </resources>
 <connections/>
</ui>
